typify/tests/schemas/reflexive.rs

```diff
@@ -1,15 +1,416 @@
+        contains: Option<Box<CoreSchemaMetaSchema>>,
+        #[serde(
+            rename = "contentEncoding",
+            default,
+            skip_serializing_if = "Option::is_none"
+        )]
+        content_encoding: Option<String>,
+        #[serde(
+            rename = "contentMediaType",
+            default,
+            skip_serializing_if = "Option::is_none"
+        )]
+        content_media_type: Option<String>,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        default: Option<serde_json::Value>,
+        #[serde(default, skip_serializing_if = "std::collections::HashMap::is_empty")]
+        definitions: std::collections::HashMap<String, CoreSchemaMetaSchema>,
+        #[serde(default, skip_serializing_if = "std::collections::HashMap::is_empty")]
+        dependencies:
+            std::collections::HashMap<String, CoreSchemaMetaSchemaObjectDependenciesValue>,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        description: Option<String>,
+        #[serde(rename = "else", default, skip_serializing_if = "Option::is_none")]
+        else_: Option<Box<CoreSchemaMetaSchema>>,
+        #[serde(rename = "enum", default, skip_serializing_if = "Option::is_none")]
+        enum_: Option<Vec<serde_json::Value>>,
+        #[serde(default, skip_serializing_if = "Vec::is_empty")]
+        examples: Vec<serde_json::Value>,
+        #[serde(
+            rename = "exclusiveMaximum",
+            default,
+            skip_serializing_if = "Option::is_none"
+        )]
+        exclusive_maximum: Option<f64>,
+        #[serde(
+            rename = "exclusiveMinimum",
+            default,
+            skip_serializing_if = "Option::is_none"
+        )]
+        exclusive_minimum: Option<f64>,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        format: Option<String>,
+        #[serde(rename = "$id", default, skip_serializing_if = "Option::is_none")]
+        id: Option<String>,
+        #[serde(rename = "if", default, skip_serializing_if = "Option::is_none")]
+        if_: Option<Box<CoreSchemaMetaSchema>>,
+        #[serde(default = "defaults::core_schema_meta_schema_object_items")]
+        items: CoreSchemaMetaSchemaObjectItems,
+        #[serde(rename = "maxItems", default, skip_serializing_if = "Option::is_none")]
+        max_items: Option<NonNegativeInteger>,
+        #[serde(rename = "maxLength", default, skip_serializing_if = "Option::is_none")]
+        max_length: Option<NonNegativeInteger>,
+        #[serde(
+            rename = "maxProperties",
+            default,
+            skip_serializing_if = "Option::is_none"
+        )]
+        max_properties: Option<NonNegativeInteger>,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        maximum: Option<f64>,
+        #[serde(rename = "minItems", default, skip_serializing_if = "Option::is_none")]
+        min_items: Option<NonNegativeIntegerDefault0>,
```

Comment:
This could be improved: perhaps
```suggestion
        #[serde(rename = "minItems", default, skip_serializing_if = "Option::is_none")]
        min_items: NonNegativeIntegerDefault0,
```
or
```suggestion
        #[serde(rename = "minItems", default = "non_neg_int_0", skip_serializing_if = "Option::is_none")]
        min_items: NonNegativeInteger,
```

Reply:
Actually this generation makes sense and is what I'd want